src/main/java/frc/robot/subsystems/drivetrain/commands/HolonomicDrive.java

```diff
@@ -0,0 +1,48 @@
+package frc.robot.subsystems.drivetrain.commands;
+
+import edu.wpi.first.wpilibj.GenericHID;
+import edu.wpi.first.wpilibj.XboxController;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.Constants;
+import frc.robot.RobotContainer;
+import frc.robot.subsystems.drivetrain.SwerveDrive;
+import frc.robot.subsystems.drivetrain.SwerveModule;
+
+/**
+ *  Get- swerveDrive, xboxController, i.
+ */
+public class HolonomicDrive extends CommandBase {
+    private final SwerveDrive swerveDrives;
+    private final XboxController xboxController = RobotContainer.xboxController;
+    private int i;
+
+    public HolonomicDrive(SwerveDrive swerveDrives, int i) {
+        this.swerveDrives = swerveDrives;
+        this.i = i;
+        addRequirements(swerveDrives);
+    }
+
+    @Override
+    public void execute() {
+        double forward = xboxController.getY(GenericHID.Hand.kLeft) * Constants.Swerve.MAX_VELOCITY;
+        double strafe = xboxController.getX(GenericHID.Hand.kRight) * Constants.Swerve.MAX_VELOCITY;
+        double rotation = xboxController.getX(GenericHID.Hand.kLeft) * Constants.Swerve.MAX_ROTATION;
```

Comment:
Maybe @adamv17 needs to decide here, but a command shouldn't know where it gets its inputs from. Use suppliers instead.